ee/codegen/src/generators/conditional-node-port.ts

```diff
@@ -172,8 +183,8 @@ export class ConditionalNodePort extends AstNode {
       doesNotContain: "does_not_contain",
       doesNotBeginWith: "does_not_begin_with",
       doesNotEndWith: "does_not_end_with",
-      null: "is_null()",
-      notNull: "is_not_null()",
+      null: isNodeOutput ? "is_nil()" : "is_null()",
+      notNull: isNodeOutput ? "is_not_nil()" : "is_not_null()",
```

Comment:
@NgoHarrison I find it odd that these look like methods here. We should probably codegen method expression calls in this case instead of saying that the parens are part of the operator name

Reply:
Agreed and good reminder for myself. Context: this was a rush from my part because I was trying to fix the original issue in a hurry and because we currently do something hacky to compose the whole port expression, I didn't know if/how difficult it would be. But looking back at it now, it probably seems like an easy fix